package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "Familia",
+  "version": "1.0.0",
+  "description": "An app to track family's required groceries",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start":"nodemon src/server.js"
```

Comment:
Your start script should include 'node' e.g. `node src/server.js`. You can use nodemon in a dev script: `"dev": "nodemon src/server.js"`. This will make pushing to heroku less hassle.